src/sourcegraph.d.ts

```diff
@@ -448,6 +839,19 @@ declare module 'sourcegraph' {
         export const rawConnection: any
     }
 
+    /**
+     * A stream of values that may be subscribed to.
+     */
+    export interface Subscribable<T> {
```

Comment:
Can this type be passed to Rx `from()`?

Reply:
The intent is to make that possible, perhaps with a helper other than `rxjs`'s `from()`. I would love your advice on how to do this. I think it would be OK to add an `@internal`-jsdoc-tagged `[Symbol.observable]: () => any` field here and mention that it's there for interop with observables/rxjs. WDYT?